snowflake_blueprints/upload_file.py

```diff
@@ -51,14 +53,43 @@ def get_args():
         choices={
             'fail',
             'replace',
-            'append'},
+            'append'
+        },
         default='append',
         required=False)
+    parser.add_argument(
+        "--snowflake-data-types", dest="snowflake_data_types", required=False
+    )
```

Comment:
For reference sake, can we include somewhere in the comments how data types should be formatted? If I'm reading it correctly, it would just be a dictionary {"column1":"column1type","column2":"column2type"}. The problem with that is that dictionaries aren't ordered.

Especially as we aim towards consolidation, I would prefer to keep consistency in our database Blueprints and use the same format as BigQuery - double nested lists. The reason for this is:
1) In BigQuery, there can sometimes be a 3rd field of MODE. Who knows if other databases will have this same need, other than column/datatype which can be cleanly mapped to key/value. It'll be easier on us to unify the parsing logic and easier on users that use multiple databases.
2) Lists keep order, so we can ensure that the table gets created with the right column order.

[["column","datatype"],["column","datatype","MODE"]]

https://github.com/shipyardapp/googlebigquery-blueprints/blob/master/googlebigquery_blueprints/upload_file.py#L212

Reply:
Also, wondering if the name should be `snowflake_schema`. That's a preference thing though.

Reply:
> Also, wondering if the name should be snowflake_schema. That's a preference thing though.
I chose to use `snowflake-data-types` because `schema` is already a different argument